html/changelogs/neural_suppressor.yml

```
@@ -0,0 +1,6 @@
+author: Vrow
+
+delete-after: True
+
+changes:
+  - bugfix: "Fixes the opearing table's neural suppressors occasionally not working."
```

```suggestion
  - bugfix: "Fixes the operating table's neural suppressors occasionally not working."
```
Feel free to ignore this suggestion.

---

FUCKING TYPOS